### PR TITLE
Remove example linked link in favor of literal

### PR DESCRIPTION
**Pull request by alfsb**
This file may need also be fully re-indented.

### CODE REVIEW

File: install/cloud/azure.xml [hunk 1]
```diff
@@ -21,8 +21,14 @@
...
-      <para>Kudu Dashboard: [your web site name].azurewebsites.net Then, the Kudu dashboard is <link xlink:href="https://your_web_site_name.scm.azurewebsites.net/">https://[your web site name].scm.azurewebsites.net/</link>. The Dashboard gives you access to some debugging capabilities, file management and site extensions. Site extensions are an Azure mechanism to add extra programs, like PHP preview builds, to your web site.
-      </para>
+     <para>
+      Kudu Dashboard: if your site has the URL
+      <literal>[site_name].azurewebsites.net</literal>, then, the Kudu dashboard
+      is <literal>https://[site_name].scm.azurewebsites.net/</literal>. The
+      Dashboard gives you access to some debugging capabilities, file management
+      and site extensions. Site extensions are an Azure mechanism to add extra
+      programs, like PHP preview builds, to your website.
+     </para>
```
Comment (jimwins) [inline install/cloud/azure.xml:31]:
```suggestion
     <para>
      Kudu Dashboard: if the website has the URL
      <literal><replaceable>site_name</replaceable>.azurewebsites.net</literal>,
      the Kudu dashboard is
      <literal>https://<replaceable>site_name</replaceable>.scm.azurewebsites.net/</literal>.
      The dashboard provides access to debugging capabilities, file
      management, and site extensions.
      Site extensions are an Azure mechanism to add extra programs,
      like PHP preview builds, to a website.
     </para>
```

**Review by jimwins — CHANGES_REQUESTED** [review]
We can delimit the variable parts of the URLs with `<replaceable>`. We don't want to re-flow a whole document at once, but there are enough wording tweaks here to warrant making this paragraph conform to preferred styles, including semantic line breaks.

**Comment by alfsb** [comment]
> We can delimit the variable parts of the URLs with `<replaceable>`. We don't want to re-flow a whole document at once, but there are enough wording tweaks here to warrant making this paragraph conform to preferred styles, including semantic line breaks.

Revision included.

I plan wait some weeks for another reviews, and then merge in case of no opposition. Editors are welcome to merge this at any time.

**Review by cmb69 — APPROVED** [review]
Thank you!

**Review by MauricioFauth — APPROVED** [review]

**Comment by jimwins** [comment]
Thanks!

**Pull request merged**